src/base/ddl_log.c

```diff
@@ -532,26 +556,54 @@ logddl_make_schema_file_name (const char *file_full_path, char *dest_path, size_
       name_tmp++;
     }
 
-  retval = snprintf (ddl_audit_handle->schema_file, buf_size, "%s_%s_%d", name_tmp, time, getpid ());
+  retval = snprintf (copy_filename, buf_size, "%s_%s_%d", name_tmp, time, getpid ());
   if (retval < 0)
     {
       assert (false);
-      ddl_audit_handle->schema_file[0] = '\0';
+      copy_filename[0] = '\0';
       return retval;
     }
 
-  retval =
-    snprintf (dest_path, buf_size, "%s/%s/%s_%s_%d", env_root, DDL_LOG_LOADDB_FILE_PATH, name_tmp, time, getpid ());
+  return retval;
+}
+
+static int
+logddl_make_copy_dir (T_APP_NAME app_name, char *copy_filename, char *copy_fullpath, size_t buf_size)
+{
+  const char *env_root = NULL;
+  char time[TIME_STRING_SIZE] = { 0 };
+  //char dest_path[PATH_MAX] = { 0 };
+  const char *name_tmp = NULL;
+  char *tpath = NULL;
```

Comment:
Variable **tpath** is defined and not used, remove it.

Reply:
Thank you, I will remove it.